core/templates/services/exploration-html-formatter.service.ts

```diff
@@ -158,6 +162,10 @@ export class ExplorationHtmlFormatterService {
       el.attr('choices', this.htmlEscaper.objToEscapedJson(
         interactionChoices));
     }
+    if (interactionChoicesD) {
+      el.attr('interactionChoicesD', this.htmlEscaper.objToEscapedJson(
+        interactionChoicesD));
```

Comment:
we will need to a new value because for shortAnswer the choices only have html, you can see this line [https://github.com/oppia/oppia/pull/12711/files#r637352952](url), but we need both html + contentId to show the correct responses. If we only display choices then it will display the correct answer order. So to solve this we need a new value which has both conentId + html.